notebook/notebookapp.py

```diff
@@ -608,6 +608,9 @@ def _webapp_settings_changed(self, name, old, new):
             help="Supply overrides for the tornado.web.Application that the "
                  "Jupyter notebook uses.")
     
+    terminado_settings = Dict(config=True,
+            help='Supply overrides for terminado. Currently only supports "shell_command".')
```

Comment:
Are there other things you envisage putting in here? Would it be simpler to have `terminals_shell_command` as a configurable thing by itself?